docs/source/cpp/compute.rst

```diff
@@ -266,10 +266,20 @@ an ``Invalid`` :class:`Status` when overflow is detected.
 +--------------------------+------------+--------------------+---------------------+
 | abs_checked              | Unary      | Numeric            | Numeric             |
 +--------------------------+------------+--------------------+---------------------+
+| acos                     | Unary      | Numeric            | Numeric             |
++--------------------------+------------+--------------------+---------------------+
 | add                      | Binary     | Numeric            | Numeric (1)         |
 +--------------------------+------------+--------------------+---------------------+
 | add_checked              | Binary     | Numeric            | Numeric (1)         |
 +--------------------------+------------+--------------------+---------------------+
+| asin                     | Unary      | Numeric            | Numeric             |
```

Comment:
I would put then in a separate table with the trigonometric kernels grouped (IMO that will be easier to keep somewhat the overview on this page)